packages/server/test/setup.ts

```diff
@@ -1,7 +1,14 @@
-import log from 'npmlog'
+import { prisma } from '@/common/prisma'
 
-import '@/common/config'
-log.level = 'silent'
+process.env.APP_SECRET_KEY = 'foo'
```

Comment:
Maybe worth a README hint to change this for improved security.

Reply:
@traumschule thank you for the review.

This value will only be used for running the integration tests. I might be missing something but I don't see how it can be a security risk. However if you still think it's the case it could be randomized and imported in the tests where it's needed.